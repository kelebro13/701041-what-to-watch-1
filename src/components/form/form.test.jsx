import renderer from "react-test-renderer";
import Form from "./form";

it(`renders property`, () => {

  const tree = renderer
    .create(<Form />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
