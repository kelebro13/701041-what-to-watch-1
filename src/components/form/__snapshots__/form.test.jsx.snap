// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders property 1`] = `
<form
  action="#"
  onSubmit={[Function]}
/>
`;
